EMalgo_continous_categorical_data.R

```diff
@@ -0,0 +1,124 @@
+#Description ####
+#EM Algorithm for continous variables and ctegorical variables
+#Model Based Learning project
+#Groupe : Florent Jakubowski et Marc Telly 
+#Date : 2020-11-15
+#For mor informations read the README
+#All functions should be tested with unit tests using testthat package
+#To understand our way of writting code see the book CleanCode. 
+
+# packages ####
+library(bayess)
+library(mvtnorm)
+
+
+# FLO function TransformCategoricalVectorToBinaryMatrice ####
+TransformCategoricalVectorToBinaryMatrice <- function (categoricalVector) {
```

Comment:
Add Description to the function.
Try to find shorter name.

Reply:
Oui comme je disais, le nommage doit rester accèssible étant donnée que c'est un package qui à pour objectif d'être utiliser par d'autre et non une boite noire propre à nous donc il faudrait trouver un compromis entre lisibilité des fonction et limite de longueur des noms des fonctions